packages/nep141-erc20/src/natural-erc20/sendToNear/findProof.js

```diff
@@ -1,7 +1,8 @@
-import Tree from 'merkle-patricia-tree'
+import { Trie } from './merkle-patricia-tree'
+// import Tree from 'merkle-patricia-tree'
+// import { promisfy } from 'promisfy'
```

Comment:
Can delete these commented lines?

Reply:
I'd like to keep them for now in case we need to go back and compare the results produced by the 2 tries via the commented functionality below